lib/reporting/identity_verification_report.rb

```diff
@@ -136,75 +158,77 @@ def merge(other)
       )
     end
 
-    def idv_final_resolution
-      data[Events::IDV_FINAL_RESOLUTION].count
+    def idv_final_resolution(data_set = data['all'])
```

Comment:
I don't think I would specify a default value for `data_set` for all these methods (i.e. force the caller to specify which data set it wants numbers for).

Reply:
Agreed